src/resdk/collection_tables.py

```diff
@@ -1,632 +1,16 @@
-                "constructor.\n"
-            )
-
-        if message:
-            raise ValueError(message)
-
-    @property
-    @lru_cache()
-    def meta(self) -> pd.DataFrame:
-        """Return samples metadata table as a pandas DataFrame object.
-
-        :return: table of metadata
-        """
-        return self._load_fetch(META)
-
-    @property
-    @lru_cache()
-    def exp(self) -> pd.DataFrame:
-        """Return expressions table as a pandas DataFrame object.
-
-        Which type of expressions (TPM, CPM, FPKM, ...) get returned depends on how the
-        data was processed. The expression type can be checked in the returned table
-        attribute `attrs['exp_type']`:
-
-        .. code-block:: python
-
-            exp = tables.exp
-            print(exp.attrs['exp_type'])
-
-        :return: table of expressions
-        """
-        exp = self._load_fetch(EXP)
-        self.gene_ids = exp.columns.tolist()
-        return exp
-
-    @property
-    @lru_cache()
-    def rc(self) -> pd.DataFrame:
-        """Return expression counts table as a pandas DataFrame object.
-
-        :return: table of counts
-        """
-        rc = self._load_fetch(RC)
-        self.gene_ids = rc.columns.tolist()
-        return rc
-
-    @property
-    @lru_cache()
-    def id_to_symbol(self) -> Dict[str, str]:
-        """Map of source gene ids to symbols.
-
-        This also gets fetched only once and then cached in memory and on disc.
-        :attr:`CollectionTables.exp` or :attr:`CollectionTables.rc` must be called
-        before this as the mapping is specific to just this data. Its intended use is
-        to rename table column labels from gene ids to symbols.
-
-        Example of use:
-
-        .. code-block:: python
-
-            exp = exp.rename(columns=tables.id_to_symbol)
-
-        :return: dict with gene ids as keys and gene symbols as values
-        """
-        species = self._data[0].output["species"]
-        source = self._data[0].output["source"]
-
-        if not self.gene_ids:
-            raise ValueError("Expression data must be used before!")
-
-        return self._mapping(self.gene_ids, source, species)
-
-    @staticmethod
-    def clear_cache() -> None:
-        """Remove ReSDK cache files from the default cache directory."""
-        clear_cache_dir_resdk()
-
-    @property
-    @lru_cache()
-    def _samples(self) -> List[Sample]:
-        """Fetch sample objects.
-
-        Fetch all samples from given collection and cache the results in memory. Only
-        the needed subset of fields is fetched.
-
-        :return: list od Sample objects
-        """
-        sample_ids = [d.sample.id for d in self._data]
-        return list(
-            self.collection.samples.filter(id__in=sample_ids, fields=SAMPLE_FIELDS)
+        """Initialize."""
+        warnings.warn(
+            "resdk.CollectionTables will be deprecated in the next major "
```

Comment:
We change major versions often. Should we report a timestamp instead? E.g., in 2022Q1.

Reply:
Good suggestion, thanks!